ui/src/containers/VolumePage.js

```diff
@@ -101,6 +104,16 @@ const VolumePage = (props) => {
     getVolumeListData(state, props),
   );
 
+  // If data has been retrieved and no volume is selected yet we select the first one
+  useEffect(() => {
+    if (volumeListData.length && !currentVolumeName) {
+      history.push({
+        pathname: `/volumes/${volumeListData[0]?.name}/overview`,
```

Comment:
```suggestion
        pathname: `/volumes/${currentVolumeName}/overview`,
```

Reply:
Not sure I understand there. We want to push to history when there is **no** currentVolumeName. In that condition currentVolumeName is undefined.

Reply:
Sorry. My bad. You're right here!